src/Helsenorge.Registries/AddressRegistry.cs

```diff
@@ -152,6 +153,15 @@ public async Task<CertificateDetails> GetCertificateDetailsForEncryptionAsync(in
                 }
 
                 certificateDetails = MapCertificateDetails(herId, certificateDetailsRegistry);
+                try
+                {
+                    certificateDetails.Certificate.Verify();
```

Comment:
For context, these were added as an attempt to fix an issue where a certificate was revoked but not in the cache. This would result in the export crashing while adding to cache.

Reply:
Vi har CertificateValidator som allerede ligger i Helsenorge.Registries, bedre å ha CertificateValidator som et optional argument til AddressRegistry ctor og kun validere sertifikatet dersom _certificateValidator != null.